### PR TITLE
Updated YAML file for MONKEY challenge (Tools & Applications)

### PR DESCRIPTION
**Pull request by LindaSt**
Updated link to the structured report on Zenodo to always point to the most recent version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by berylrab — APPROVED** [review]

**Comment by LindaSt** [comment]
@berylrab We'll update some data later today. Can you please wait until then to merge it? I'll comment here when it's done.

**Comment by berylrab** [comment]
Hi Linda, if you are talking about data being updated directly in the bucket, the changes are reflected immediately.
Any metadata changes submitted through a pull request will be reflected when we merge the pull request.

**Pull request merged**
